cpp/open3d/visualization/visualizer/O3DVisualizer.h

```diff
@@ -175,6 +175,7 @@ class O3DVisualizer : public gui::Window {
     void SetLineWidth(int line_width);
     void EnableGroup(const std::string& group, bool enable);
     void SetMouseMode(gui::SceneWidget::Controls mode);
+    void SetCollapsableVertOpen(const std::string& name, bool open);
```

Comment:
nit: consider naming `SetPanelOpen` instead.

Reply:
Interestingly, `SetPanelOpen` is exactly what I originally named the function. I did change it because the source code seems to refer the "Panel" as the setting panel, and "Verts" as the elements within the panel. This is not absolute though, different places uses different names. I don't know if we have any definitions to the UI elements but I strongly think that it should probably exists.
For the mean time, I'll happily change the name if you think it's clearer for the user/ coders.

Reply:
Updated in https://github.com/isl-org/Open3D/pull/6865/commits/459c7ab7e9b369a9daba4491e3e2edb2f85c82b9